code/modules/power/tesla/coil.dm

```diff
@@ -89,13 +100,22 @@
 	anchored = 0
 	density = 1
 
+	can_buckle = TRUE
```

Comment:
How exactly are the meant to get buckled to something that's density = 1? You can't move people on top of it, it acts as a wall. If the answer is 'check_loc = FALSE', my testing suggests that doesn't work.